specs/cm/change-mgt-spec.html

```diff
@@ -1115,6 +980,35 @@ <h2>Acknowledgements</h2>
           Carol Geyer, OASIS<br />
         </p>
       </section>
+
+      <section id="versionCompability" class="appendix">
+        <h2>Errata</h2>
+        <p>
+          OSLC Change Management version 3.0 deprecated a number of properties that refer to OSLC Quality Management
```

Comment:
Just this line I just commented on states that "OSLC Change Management version 3.0 **deprecated** a number of properties ..." 
That is, the properties oslc_qm:relatedChangeRequest, oslc_qm:testsChangeRequest, ...

Reply:
I got it now. How about this restructing, because the important inforation is comming too late. that is, move that last new pargraph early.

`OSLC Change Management version 3.0 incorrectly deprecated a number of properties that refer to OSLC Quality Management artifacts. With the introduction of OSLC Configuration Management 1.0, and that OSLC Change Management resources are not versioned resources, these properties will now be required to create links between unversioned Change Management resources and versioned Quality Management resources. Therefore these properties are no longer deprecated, and their ResourceShapes have been added to the constraints.`